subsys/bluetooth/shell/gatt.c

```diff
@@ -699,22 +699,22 @@ static uint8_t print_attr(const struct bt_gatt_attr *attr, uint16_t handle,
 
 static int cmd_show_db(const struct shell *sh, size_t argc, char *argv[])
 {
-	struct bt_uuid_16 uuid;
+	struct bt_uuid_16 uuid_16;
```

Comment:
I think we could give this variable a semantically meaningful name. "attr_type_query"?

It would also be great if you moved the declaration to above `num_matches`. This variable has a bigger scope than it needs.

Reply:
Let’s not start expanding the scope of this PR. It should be about fixing the badly named global variables that cause shadowing warnings. Local variable naming is more of a style issue that isn’t the actual cause of compiler warnings in this case. 

Generally, local variables used in a function can have short general names, if it’s clear where and how they’re used (eg in this case when the usage is close to the declaration and there are no other UUIDs the function deals with). The variable could be called `attr_type` here, but it’s an unrelated matter to fixing the compiler warning.